.github/workflows/ci.yml

```diff
@@ -624,7 +624,8 @@ jobs:
       - docs
       - docs-spell-check
     if: >
-      (github.ref == 'refs/heads/master' || github.ref == 'refs/heads/v1-10-test') &&
+      (github.ref == 'refs/heads/master' || github.ref == 'refs/heads/v1-10-test' ||
+      github.ref == 'refs/heads/v2-0-test') &&
```

Comment:
There is no need to add it here either. The effect of this change is that it will push images to githuib registry. We cannot have separate images (master/v2-0-test) for those two because this  is not "Rebase-friendly" we would have to keep a different branch name in the code for v2-0-test (as we keep _initialisation.sh (``export DEFAULT_BRANCH="master" or "DEFAULT_BRANCH="v1-10-test").

If we want to continue rebasing rather than diverging, this has to remain  "master" in both branches. Then the master push and rebasing v2-0-test would effectively rebuild and re-push the same image.